Userloginpage.php

```diff
@@ -0,0 +1,79 @@
+<?php
+require 'partials/nav.php';
+
+?>
+
+<?php
+ $_SESSION['loggedin']=false;
+if(isset($_POST['login'])){
+
+    include 'partials/db_connect.php';
+
+    $email = $_POST['email'];
+    $pass = $_POST['password'];
+    $sql="SELECT * FROM `user_details` WHERE email='$email'";
+    
+    $result = mysqli_query($link,$sql);
+    $num = mysqli_num_rows($result);
+    if($num==1){
+        while($row=mysqli_fetch_assoc($result)){
+            if(password_verify($pass,$row['password'])){
+                $login= true;
+                session_start();
+                $_SESSION['loggedin']=true;
+                $_SESSION['email']=$email;
+                echo "done";
+                header('location:Userhomepage.php');
+            }
+        }
+      
+    }
+}
+
+
+echo '
```

Comment:
You cannot echo a whole HTML code, Thats not a good practice. Find alternative ways to fix this.